src/main/java/io/fabric8/maven/docker/model/ContainerDetails.java

```diff
@@ -77,10 +77,7 @@ public Map<String, String> getLabels() {
     public String getName() {
         String name = json.get(NAME).getAsString();
 
-        if (name.startsWith(SLASH)) {
-            name = name.substring(1);
-        }
-        return name;
+        return name.startsWith(SLASH)?name.substring(1):name;
```

Comment:
I find if statements more readable than ternary operators.

If you're using ternary here, please add a bit formatting so that it's more readable
```suggestion
        return name.startsWith(SLASH) ? name.substring(1) :
          name;
```